core/src/test/java/org/infinispan/test/AbstractInfinispanTest.java

```diff
@@ -57,7 +57,7 @@
       boolean isSatisfied() throws Exception;
    }
 
-   protected static final Log log = LogFactory.getLog(MethodHandles.lookup().lookupClass());
+   protected final Log log = LogFactory.getLog(getClass());
```

Comment:
Hrmm is this an issue with the other JIRA that changed this @tristantarrant ?

Reply:
@wburns The problem is that logs of tests use the inherited field, and their messages end up logged with the `AbstractInfinispanTest` category. I initially wanted to make it private static, but there were too many tests using it and I wouldn't have been able to sneak it into this PR that way :)